scantools/run_navvis_to_capture.py

```diff
@@ -54,11 +54,10 @@ def convert_to_us(time_s):
 
 def run(input_path: Path, capture: Capture, tiles_format: str, session_id: Optional[str] = None,
         downsample_max_edge: int = None, upright: bool = True, export_as_rig: bool = False,
-        copy_pointcloud: bool = False):
+        export_trace: bool = False, copy_pointcloud: bool = False):
 
     if session_id is None:
         session_id = input_path.name
-    assert session_id not in capture.sessions
```

Comment:
Intended change?

Reply:
Actually yes, you can overwrite a session that was already created, so I don't have to remove it and start always from scratch, saves a lot of time especially while debugging.